index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+      integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css"
+    />
+  </head>
+  <body>
+    <section class="header">
+      <div class="logo">Felix</div>
+      <div class="menu"><i class="bi bi-list"></i></div>
+      </div>
+    </section>
+    <section class="headline">
+        <div class="name">
+            <h2>
+                Hey There.I’m Felix
+            </h2>
+        </div>
+        <div class="name-2"><h3>I’am a Software Developer</h3></div>
+        <div class="p-name">
+            <p>
+                I can help you build a product, 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+        </div>
+
+        <section class="social-illustration">
+
+            <div class="socialmedia">
+                <a href="#"><img src="assets/Vector.png" alt=""/></a>
+                <a href="#"><img src="assets/Group.png" alt=""/></a>
+                <a href="#"><img src="assets/Vector2.png" alt=""/></a>
+                <a href="#"><img src="assets/Vector3.png" alt=""/></a>
+                <a href="#"><img src="assets/Vector4.png" alt=""/></a>
+    
+                <div class="diagram">
+                    <img src="assets/illustration.png" alt=""/>
+                </div>
```

Comment:
- [ ] Since this image is not part of the content, kindly  use CSS background-image property and ensure that it is properly aligned. For more info you can check [this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image). 😄